app/models/connect_four.rb

```diff
@@ -0,0 +1,168 @@
+class ConnectFour
+  attr_reader :player_id
+  attr_accessor :game_board, :board, :message
+
+  def initialize (move, board=nil)
+    @message = ""
+    @board = board
+    player_move = move.to_i - 1
+    insert_coin(player_move, "X") #human move
+    @message = set_message
+    insert_coin(pick_randomly, "O")
+    unless @message == "Congrats! You win!" || @message == "Board is full. You draw!"
+      @message = ai_wins
+    end
+  end
```

Comment:
This seems like a lot of instance variables for an object that's not going to persist very long. Why not make `message` it's own message that wraps up all the methods editing your instance variable?